torchao/experimental/quant_api.py

```diff
@@ -463,3 +463,56 @@ def quantize(self, model: nn.Module) -> nn.Module:
             },
         )
         return model
+
+
+from torchao.experimental._linear_8bit_act_xbit_weight_layout import Linear8BitActXBitWeightLayout
+from torchao.quantization.quant_api import (
+    _get_linear_subclass_inserter,
+    MappingType,
+    to_affine_quantized_intx,
+    ZeroPointDomain,
+)
+
+
+def int8_dyn_act_intx_weight(
```

Comment:
we are spelling out all the words in our API so far, so this should be `int8_dynamic_activation_intx_weight` I think